__tests__/data/proxy_output.json

```diff
@@ -108,5 +108,18 @@
         "status": 204
       }
     }
+  },
+  {
+    "output": {
```

Comment:
Don't we have a test for testing the destination request payload?
Even if we had this test previously, we wouldn't have caught the error we saw.

Reply:
> Even if we had this test previously, we wouldn't have caught the error we saw.

Yes we wouldn't have caught this. 

> Don't we have a test for testing the destination request payload?

Currently no